upgrade/runner.py

```diff
@@ -135,13 +133,10 @@ def product_upgrade_capsule(cap_host):
             with LogAnalyzer(cap_host):
                 current = execute(get_sat_cap_version, 'cap', host=cap_host)[cap_host]
                 if settings.upgrade.from_version != settings.upgrade.to_version:
-                    execute(satellite_capsule_upgrade,
-                            cap_host, sat_host, host=cap_host)
-                elif settings.upgrade.from_version == settings.upgrade.to_version:
-                    execute(satellite_capsule_zstream_upgrade,
-                            cap_host, host=cap_host)
-                upgraded = execute(
-                    get_sat_cap_version, 'cap', host=cap_host)[cap_host]
+                    execute(satellite_capsule_upgrade, cap_host, sat_host, host=cap_host)
+                else:
+                    execute(satellite_capsule_upgrade, cap_host, True, host=cap_host)
```

Comment:
In else clause `sat_host = True` ?  I don't think this is correct

Reply:
@lpramuk Good catch! updated, request to re-review.
Thank you :)